test/Microsoft.TestPlatform.AcceptanceTests/Properties/AssemblyInfo.cs

```diff
@@ -23,4 +23,4 @@
 [assembly: Guid("755996fa-672a-4272-9776-7f707a520058")]
 
 // Enable IAP at class level with as many threads as possible based on CPU and core count.
-[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.ClassLevel)]
+[assembly: Parallelize(Workers = 0, Scope = ExecutionScope.MethodLevel)]
```

Comment:
Is it faster?

Reply:
It is not, MethodLevel is the correct level for us, but if you remember we had quite a few issues with it being unstable for our tests. So we moved back to class, but that makes running the matrix tests annoying be cause they will run one after each other instead in parallel.
Reverted this change.